### ISSUE TITLE
Is there a way to edit UTT (total usage hours) on an LG C1?

### ISSUE THREAD

**Issue by DitherMan420** [Maassoft/ColorControl#221]
So I am an idiot and did an INSTOP on the LG C1. Everything is fine but it cleared the UTT back to 0 hours. Is there a way to edit the UTT with color control to enter the correct hours? This is important for the 2000 hour pixel cleaning that these panels do. i can always trigger it manually at 1800 hours instead of 2000 (i had 200 hours on the TV) but I was wondering if UTT can be changed with this program? If so, that would be fantastic!

**Comment by Maassoft** [comment]
No, I don't think it's possible. I've scanned the firmware files to check for services/end points but can't find a way to edit the total power on hours. Even if there was, it would probably be a private (internal) end point and would require a rooted TV. 
Btw, I wouldn't worry about +/- 200 hours if I were you. My C9 is over 10000 hours and according to the service menu only had 4 major pixel cleaning events.

**Comment by DitherMan420** [comment]
> No, I don't think it's possible. I've scanned the firmware files to check for services/end points but can't find a way to edit the total power on hours. Even if there was, it would probably be a private (internal) end point and would require a rooted TV. Btw, I wouldn't worry about +/- 200 hours if I were you. My C9 is over 10000 hours and according to the service menu only had 4 major pixel cleaning events.

Okay no problem. Yes, I feel better about it now. I learned that the pixel refresher/cleaner cycles don't actually prevent burn in. All they do is help with uniformity. So they actually "hide" burn-in. Burn-in will happen regardless of whether you run pixel cycles. It all just depends on how you use the TV.

**Comment by Amoeba00** [comment]
That's good to know that the INSTOP clears the UTT - but at least the regular reset to default does not.

**Comment by DitherMan420** [comment]
> Didn't realize that the INSTOP clears the UTT (of course, now googling that it's obvious), but at least the regular reset to default does not. To be clear, CC doesn't have the INSTOP button available and you'd need the actual service remote for that, correct?

I am not sure if Color Control has the INSTOP button actually. That is a good question. Check it out and see, and let me know!

**Comment by Amoeba00** [comment]
If it does, it's not obvious - that's why I was asking.

**Comment by rippingitonetime** [comment]
Does it have the instop button ???
